cypress/fixtures/dashboard/opensearch_dashboards/data_with_index_pattern.json.txt

```diff
@@ -0,0 +1,65 @@
+{
+  "type": "doc",
+  "value": {
+    "id": "index-pattern:timestamp-*",
```

Comment:
is this the behavior we wanna continue? i think we did this from the legacy but if we we want to add more documents to these tests to capture the cases we are running into when regressions occur

Reply:
if not then we should just consider having stored the ndjson of saved objects and then we just upload them on test start. 

this way if we have massive test data we dont know need to open or modify that file to modify the saved objects